recipes/aravis/all/conanfile.py

```diff
@@ -72,8 +66,8 @@ def layout(self):
 
     def requirements(self):
         # glib-object.h and gio/gio.h are used in several public headers
-        self.requires("glib/2.78.1", transitive_headers=True)
-        self.requires("libxml2/2.12.3")
+        self.requires("glib/2.78.3", transitive_headers=True)
```

Comment:
We're looking into version ranges for `glib`, as right now half its dependants conflict with eachother, and is looking promising for now, sorry for the delay on getting this one thru while we review that :)

Reply:
There are now 12 recipes using 2.78.3 and 7 using 2.78.1, so even if not using version ranges yet for glib, it looks we can move forward this PR